src/flcl-util-kokkos-f.f90

```diff
@@ -159,20 +158,17 @@ end subroutine kokkos_finalize
 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 !!! kokkos library helper routine implementations
 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
-  subroutine kokkos_print_configuration( prepend_name_in, file_name_in )
+  subroutine kokkos_print_configuration( file_name_in )
```

Comment:
How useful is that?
Is there an abstraction similar to an output stream in Fortran?

Reply:
> Is there an abstraction similar to an output stream in Fortran?

I couldn't find anything that maps to C++ output stream but I don't know Fortran...

Reply:
Let's keep it for now